evals/registry/solvers/fixie.yaml

```diff
@@ -3,3 +3,9 @@ generation/direct/ultravox-v0.2:
   args:
     completion_fn_options:
       model: fixie-ai/ultravox-v0.2
+
+generation/direct/gpu:
```

Comment:
I think I'd make this something like `generation/gpu/ultravox-dev` in the event we want to easily swap models.

Reply:
Q: is there a way to provide the `model` argument from CLI instead of hard-coding it here?

Reply:
eg `--completion_args=model=gpt-4o`